packages/meta-harness/lib/isolation.py

```
@@ -0,0 +1,566 @@
+            if _paths_intersect(allowed, blocked):
+                raise IsolationError(
+                    f"proposer.isolation allowRead path intersects forbidden asset: "
+                    f"{allowed} <-> {blocked}"
+                )
+
+
+def _allow_write_paths(view_dir: Path, ephemeral_home: Path, proposer_tool: str) -> list[Path]:
+    paths = [view_dir]
+    if proposer_tool == "codex":
+        paths.append(ephemeral_home)
+    paths.extend([Path("/tmp"), Path("/private/tmp")])
```

Ruff (0.15.20)

[error] Probable insecure usage of temporary file or directory: "/tmp"

(S108)